src/config/Config.ts

```diff
@@ -117,6 +119,27 @@ export const sortedIntegrationList = (
   );
 };
 
+export const defaultIconList = {
+  [IntegrationCategory.COMMUNICATIONS]: <IntegrationIconTypes>{
+    [UserIntegrationType.GOOGLE_CHAT]: <IntegrationIcon>{
+      name: 'gchat',
```

Comment:
This name should be `IntegrationType.GOOGLE_CHAT`
```suggestion
      name: IntegrationType.GOOGLE_CHAT,
```